src/connector/src/sink/kafka.rs

```diff
@@ -106,6 +106,11 @@ pub struct KafkaConfig {
         deserialize_with = "deserialize_bool_from_string"
     )]
     pub use_transaction: bool,
+
+    /// We have parsed the primary key for an upsert kafka sink into a `usize` vector representing
+    /// the indices of the pk columns in the frontend, so we simply store the primary key here
+    /// as a string.
+    pub primary_key: Option<String>,
```

Comment:
What if there are multiple pks

Reply:
This is the raw string by which users define the primary key. For example:

```SQL
create sink si from t with (
   ......,
   primary_key = 'v1, v2'
);
```

Then `primary_key` here will be `Some("v1, v2".to_string())`.